sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -79,6 +81,8 @@ class DataflowRunner(PipelineRunner):
       CreatePTransformOverride(),
   ]
 
+  _STAGED_PIPELINE_FILENAME = "pipeline.pb"
```

Comment:
<!--new_thread; commit:d9ce45cc29f40f0d0a656dd0333423d18491e4df; resolved:0-->
Could you move this to runners/dataflow/internal/names.py ?

Reply:
Done